src/solution/my_edge_app/docker-compose_Edge.yml

```diff
@@ -26,23 +26,32 @@ services:
 
 ##### INFLUXDB ######
   influxdb:
-    image: influxdb:1.7-alpine  # Define image to pull from docker hub if not already on your machine available
+    image: influxdb:2.2-alpine          # Define image to pull from docker hub if not already on your machine available
     container_name: influxdb            # Name of the influx-db container
-    restart: unless-stopped             # always restarts (see overview page 12 Industrial Edge Developer Guide)  
-    mem_limit: 1400m
-    environment:                        # Environment variables available at container run-time
-      INFLUXDB_DB: edgedb         # Variable of INFLUXDB_DB will be set at runtime as well  ### Hardcoded of ENV
-      INFLUXDB_DATA_INDEX_VERSION: 'tsi1'
+    environment:                        
+      INFLUXD_LOG_LEVEL: debug
+    restart: unless-stopped             # always restarts (see overview page 12 Industrial Edge Developer Guide) 
+    mem_limit: 350m
     logging:                            # allow logging
       options:                          # we use best pactice here as limiting file size and rolling mechanism
         max-size: "10m"                 # File size is 10MB
         max-file: "2"                   # only 2 files created before rolling mechanism applies
-      driver: json-file
+      driver: json-file  
     volumes:                            # mount volume from host
       - db-backup:/var/lib/influxdb     # mount named volume 'db-backup' to host's path to /var/lib/influxdb
+    healthcheck:
+      test: "exit 0"
+    networks:                           # define networks connected to container 'influxdb' 
+      proxy-redirect:                   # Name of the network
+  
+  init-db:
+    image: init-db:v0.0.1               # Docker Init Container to Setup the Main InfluxDB v2.2 Container
+    mem_limit: 100m
```

Comment:
Please add build statement with a path https://docs.docker.com/compose/compose-file/compose-file-v2/#build 
the goal is that user of the guide can just directly run docker-compose up -d commad after runnign the  [mosquite docker copose file](src/solution/mqtt_broker_mosquitto/docker-compose.yml)